data/json/item_groups.json

```diff
@@ -1162,6 +1167,7 @@
       [ "kimono", 2 ],
       [ "yukata", 6 ],
       [ "haori", 2 ],
+      [ "foodperson_mask", 1 ],
```

Comment:
Why is foodperson mask in an alcoholic dream PR?

Reply:
It was to bring it up to date with current commits, because I couldn't think of another way to bring my fork up to date without problematic "commits to (fork) master" other than to make a new one. I then proceeded to copy over the files I changed to the new branch, which overwrote current commits.

Reply:
Oh, I should remove that one line.